docs/docker_container.md

```diff
@@ -485,10 +485,10 @@ Below is a config example using heterogeneous plugin with GPU as a primary devic
 
 ## Security Considerations <a name="sec"></a>
 
-OpenVINO Model Server docker containers, by default, starts with the security context of local account ovms with linux uid 5000. It ensure docker container has not elevated permissions on the host machine. This is in line with best practices to use minimal permissions to run docker applications. You can change the security context by adding --user parameter to docker run command. It might be needed for example to load mounted models with restricted access. For example:
+OpenVINO Model Server docker containers, by default, starts with the security context of local account ovms with linux uid 5000. It ensure docker container has not elevated permissions on the host machine. This is in line with best practices to use minimal permissions to run docker applications. You can change the security context by adding --user parameter to docker run command. It might be needed for example to load mounted models with restricted access. You should also consider adding a --read-only flag to force a container's process to only write to locations that will be persisted with volumes. For example:
```

Comment:
```suggestion
OpenVINO Model Server docker containers, by default, starts with the security context of local account ovms with linux uid 5000. It ensure docker container has not elevated permissions on the host machine. This is in line with best practices to use minimal permissions to run docker applications. You can change the security context by adding --user parameter to docker run command. It might be needed for example to load mounted models with restricted access. 
For additional security hardening, you might also consider preventing write operations on the container root filesystem by adding a --read-only flag. It might prevent undesired modification of the container files. It case the cloud storage is used for the models (S3, GoogleStorage or Azure storage), restricting root filesystem should be combined with  `--tmpfs /tmp` flag.
```